### ISSUE TITLE
Dataset for each model

### ISSUE THREAD

**Issue by VanitarNordic** [flyyufelix/cnn_finetune#9]
Hello Sir,

Thank you for the repo. 

How can I prepare the dataset for each model?
you know I mean it is not quite clear what format they have followed or used the model to do classification or semantic segmentation (for example VOC or similar), therefore the way of creating the dataset and feeding is unknown.

**Comment by ahundt** [comment]
These are converted pretrained weights from other platforms. To train your own on imagenet with tensorflow I suggest https://github.com/ppwwyyxx/tensorpack or https://www.tensorflow.org/performance/performance_models

**Comment by flyyufelix** [comment]
These are models pretrained on ImageNet. If you want to train say a classifier on your own dataset, using pretrained model usually lead to faster converge and better accuracy. For more information about how to use the pretrained model feel free to check out my blog post https://flyyufelix.github.io/2016/10/03/fine-tuning-in-keras-part1.html

If what you are looking for is the format of the dataset to be consumed by the model, you can refer to this thread: https://github.com/flyyufelix/cnn_finetune/issues/4
